src/utils/asyncPopulate.js

```diff
@@ -12,8 +12,8 @@ export default function asyncPopulate(target, source) {
     if (Array.isArray(source[attr])) {
       target[attr] = [];
       promise = asyncPopulate(target[attr], source[attr]);
-    } else if (source[attr] === null) {
-      target[attr] = null;
+    } else if (source[attr] == null) {
+      target[attr] = source[attr];
```

Comment:
Rather than using `==`, add another condition that makes it clear what you're testing for. I'm guessing you want `else if (source[attr] === null || source[attr] === undefined`.

Reply:
Sure thing. Shouldn't `null` still come thru as `null` rather than `undefined` tho?